src/task/work.ts

```diff
@@ -195,40 +202,40 @@ export class WorkContext {
    * @param options Additional run options.
    */
   async transfer(from: string, to: string, options?: CommandOptions): Promise<Result> {
-    this.logger.debug(`WorkContext: transfering ${from} to ${to}`);
+    this.logger.debug(`Transferring ${from} to ${to}`);
```

Comment:
We might push `from` and `to` into context so that the template will not be always "stringified" by V8. Same remark to the other places where the teamplate is used. That's probably a minor performance improvement.